[BUG] Cocogitto-action crash on cocogitto 5.6.0 config file

Hi everyone,

I installed cocogitto 5.6.0 and did a `cog init` in my project.
Here  the generated file :

```toml
from_latest_tag = false
ignore_merge_commits = false
generate_mono_repository_global_tag = true
branch_whitelist = []
skip_untracked = false
pre_bump_hooks = []
post_bump_hooks = []
pre_package_bump_hooks = []
post_package_bump_hooks = []

[git_hooks]

[commit_types]

[changelog]
path = "CHANGELOG.md"
authors = []

[bump_profiles]

[packages]
```

And now my github action is crashing with the following error:

```shell
Setting git user : cog-bot
Settings git user email cog@demo.org
Error: failed to parse config

    cause unknown field `skip_untracked`, expected one of `ignore_merge_commits`, `branch_whitelist`, `tag_prefix`, `pre_bump_hooks`, `post_bump_hooks`, `commit_types`, `changelog`, `bump_profiles`

Error: Process completed with exit code 1.
```

I suggest that cocogitto-action must ignore the new field or be updated to use cocogitto 5.6.0 😄

--- a/install.sh
+++ b/install.sh
@@ -1,12 +1,12 @@
 #!/bin/sh
 
 CUR_DIR=$(pwd)
-VERSION=5.6.0
+VERSION=6.0.1
 TAR="cocogitto-$VERSION-x86_64-unknown-linux-musl.tar.gz"
 BIN_DIR="$HOME/.local/bin"
 
 mkdir -p "$BIN_DIR"
 cd "$BIN_DIR" || exit
 curl -OL https://github.com/cocogitto/cocogitto/releases/download/"$VERSION"/"$TAR"
-tar xfz $TAR
+tar --strip-components=1 -xzf $TAR x86_64-unknown-linux-musl/cog
 cd "$CUR_DIR" || exit

